src/components/Buildbaron/BuildBaronTable.tsx

```diff
@@ -1,18 +1,14 @@
 import React from "react";
-import styled from "@emotion/styled/macro";
-import Badge from "@leafygreen-ui/badge";
-import { uiColors } from "@leafygreen-ui/palette";
-import { Subtitle } from "@leafygreen-ui/typography";
 import { Table } from "antd";
-import { StyledLink } from "components/styles";
-import { BuildBaron, JiraTicket } from "gql/generated/types";
-import { getDateCopy } from "utils/string";
-
-const { blue, gray } = uiColors;
+import { GetCreatedTicketsQuery } from "gql/generated/types";
+import { JiraTicketRow } from "./BBComponents";
 
 const columns = [
   {
-    render: (text: string, { key, fields }: JiraTicket): JSX.Element => (
+    render: (
+      text: string,
+      { key, fields }: GetCreatedTicketsQuery["bbGetCreatedTickets"][0]
```

Comment:
Can you extract this type to a variable just so its easier to understand.

`type CreatedTicket = GetCreatedTicketsQuery["bbGetCreatedTickets"][0]`